### PR TITLE
Allow margin for p in markers

### PR DESCRIPTION
**Pull request by weppos**
Otherwise we are forced to use HTML as in https://github.com/dnsimple/dnsimple-support/pull/702

This rule was added in https://github.com/dnsimple/dnsimple-support/pull/451 but I cannot find exactly why we needed it (it is incorporated in a bunch of other changes).

Also this rule doesn't exist on the developer site, and I'm confident the style for the markers could/should be the same
https://github.com/dnsimple/dnsimple-developer/blob/master/content/assets/css/_application.scss
In fact, should we sync the marker definition between these two projects?

Screenshot 2021-01-04 at 15 22 44

### CODE REVIEW

File: content/assets/css/_application.scss [hunk 1]
```diff
@@ -24,9 +24,6 @@
   h4 {
     margin-top: 0px;
   }
```
Comment (dallasread) [inline content/assets/css/_application.scss:26]:
```suggestion
  }
  
  :last-child {
    margin: 0;
  }
```

Comment (dallasread) [inline content/assets/css/_application.scss:26]:
```suggestion
  }
  
  :last-child {
    margin-bottom: 0;
  }
```

**Review by dallasread — APPROVED** [review]
👍 LGTM. I left a suggestion that ¿_may_? align more with the intention of the existing code.

It had not occured to me that we were trying to keep the repos somewhat in sync. 😬

File: content/assets/css/_application.scss (hunk 1 above)
Comment (sbastn) [reply content/assets/css/_application.scss:26]: ✅ applied suggestion

Comment (sbastn) [reply content/assets/css/_application.scss:26]: Correct, applied suggestion

**Review by sbastn — APPROVED** [review]

**Comment by weppos** [comment]
Thanks.

**Pull request merged**